app/services/foreman/telemetry_helper.rb

```diff
@@ -23,10 +23,12 @@ def telemetry_duration_histogram(name, scale = 1000, tags = {})
     when :min, :minutes
       scale = 1 / 60
     end
-    before = Time.now.to_f
+    before = Process.clock_gettime(Process::CLOCK_MONOTONIC)
     yield
   ensure
-    after = Time.now.to_f
-    telemetry_observe_histogram(name, (after - before) * scale, tags)
+    after = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+    duration = (after - before) * scale
+    telemetry_observe_histogram(name, duration, tags)
+    return duration
```

Comment:
As the linter indicate, you do not need to do an explicit return here, because the last statement of ruby always returned as a value., so using `duration` as last statement will return it's value.

Reply:
This is an `ensure` block:

```
>> def test
>>   1
>> ensure
?>   2
>> end

>> puts test
1
```

Reply:
I'ved learned something new, thanks :)